src/main/play-doc/operation/Install.md

```diff
@@ -255,19 +259,112 @@ On some Debian distributions you may need to add a dedicated Personal Package Ar
+
+#### Prerequisite
+
+Docker needs to be installed on the host where HAProxy docker container will run.
+
+#### Preparing HAProxy config
+
+Setup basic HAProxy configuration for the Docker based HAProxy. Ensure the `conductr-agent` user which runs the ConductR-HAProxy has ownership to the HAProxy config file.
 
 ```bash
+[172.17.0.1]$ sudo mkdir /etc/haproxy
+[172.17.0.1]$ sudo touch /etc/haproxy/haproxy.cfg
 [172.17.0.1]$ sudo chown conductr-agent:conductr-agent /etc/haproxy/haproxy.cfg
 ```
 
+Populate the `/etc/haproxy/haproxy.cfg` file with the following configuration.
+
+```
+defaults
+    log global
+    mode    http
+    option  httplog
+    option  dontlognull
+    timeout connect 5000
+    timeout client  50000
+    timeout server  50000
+
+frontend monitor
+  bind :65535
+  mode http
+  monitor-uri /test
+```
+
+#### Running the HAProxy Docker container
+
+Run the HAProxy Docker container.
+
+```
+[172.17.0.1]$ docker run -d --name haproxy -p 10.0.7.118:9000:9000 -p 10.0.7.118:9999:9999 -p 10.0.7.118:65535:65535 -v /etc/haproxy/haproxy.cfg:/usr/local/etc/haproxy/haproxy.cfg haproxy:1.5
+```
+
+The container has `haproxy` as its name.
+
+The proxy configuration located at `/etc/haproxy/haproxy.cfg` is mounted within the container on `/usr/local/etc/haproxy/haproxy.cfg`.
+
+The container exposes the port `9000`, `9999`, and `65535` to expose default port for ConductR HTTP-based endpoints, Visualizer bundle, and HAProxy test endpoint. Additional ports for your endpoints can be exposed through the `-p` option as required.
+
+The ports exposed by the container is bound to the `10.0.7.118` address as it's the interface where external traffic is expected to arrive from. Substitute the `10.0.7.118` address with the correct address in your environment.
+
+Run `docker ps -a` to ensure the container has been started successfully.
+
+```
+[172.17.0.1]$ docker ps -a
+CONTAINER ID        IMAGE               COMMAND                CREATED             STATUS              PORTS                                                                               NAMES
+534236e89eac        haproxy:1.5         "/docker-entrypoint.   15 seconds ago      Up 15 seconds       10.0.7.118:9000->9000/tcp, 10.0.7.118:9999->9999/tcp, 10.0.7.118:65535->65535/tcp   haproxy
+```
+
+Additional check can be performed by using `curl` command against the HAProxy test endpoint. Substitute the address `10.0.7.118` with the ip address appropriate to your environment.
+
+```
+[172.17.0.1]$ curl -v http://10.0.7.118:65535/test
+*   Trying 10.0.7.118...
+* Connected to 10.0.7.118 (10.0.7.118) port 65535 (#0)
+> GET /test HTTP/1.1
+> Host: 10.0.7.118:65535
+> User-Agent: curl/7.48.0
+> Accept: */*
+>
+* HTTP 1.0, assume close after body
+< HTTP/1.0 200 OK
+< Cache-Control: no-cache
+< Connection: close
+< Content-Type: text/html
+<
+<html><body><h1>200 OK</h1>
+Service ready.
+</body></html>
+* Closing connection 0
+```
```

Comment:
Otherwise there will be no way to check if the HAProxy itself is running fine within the container.

I've done this when setting up the HAProxy within our DCOS cluster, I find it useful to have, and so I'd thought it would be useful to include it.

I'm happy to remove it though.